Doc/library/uuid.rst

```diff
@@ -12,8 +12,8 @@
 
 This module provides immutable :class:`UUID` objects (the :class:`UUID` class)
 and the functions :func:`uuid1`, :func:`uuid3`, :func:`uuid4`, :func:`uuid5`,
-and :func:`uuid.uuid8` for generating version 1, 3, 4, 5, and 8 UUIDs as
-specified in :rfc:`9562` (which supersedes :rfc:`4122`).
+:func:`uuid6`, and :func:`uuid8` for generating version 1, 3, 4, 5, 6,
+and 8 UUIDs as specified in :rfc:`9562` (which supersedes :rfc:`4122`).
```

Comment:
After whichever PR you merge first, we can make this easier to read, rather than having to scan the full list:
```suggestion
:func:`uuid6`, and :func:`uuid8` for generating version 1 and 3-8
UUIDs as specified in :rfc:`9562` (which supersedes :rfc:`4122`).
```

Reply:
I think I'll merge first UUIDv6 followed by UUIDv7. So I'll do the modification on the UUIDv7 PR.